src/Commands/GraphicField.php

```diff
@@ -81,6 +81,9 @@ public function encodeImage(string $image, int $width, bool $compressData = true
         if ($im === false) {
             throw new Exception('Image not supported');
         }
+        if ($width <= 0) {
+            $width = imagesx($im);
+        }
 
         $aux = imagescale($im, $width);
         $height = imagesy($aux);
```

Comment:
@erikn69 does it make sense not rescale the image again if the intention is to use its original size?

```suggestion
        if ($width <= 0) {
            $width = imagesx($im);
            $height = imagesy($im);
        } else {
            $aux = imagescale($im, $width);
            $height = imagesy($aux);
        }
```

Reply:
Yes, I just avoided changing the code too much.